test/integration/BPool.t.sol

```diff
@@ -53,19 +58,27 @@ abstract contract PoolSwapIntegrationTest is Test, GasSnapshot {
 
     factory = _deployFactory();
 
+    vm.startPrank(lp);
+    pool = factory.newBPool();
+
     deal(address(dai), lp, DAI_LP_AMOUNT);
     deal(address(weth), lp, WETH_LP_AMOUNT);
+    deal(address(wbtc), lp, WBTC_LP_AMOUNT);
 
     deal(address(dai), swapper.addr, DAI_AMOUNT);
     deal(address(weth), swapperInverse.addr, WETH_AMOUNT_INVERSE);
 
-    vm.startPrank(lp);
-    pool = factory.newBPool();
+    deal(address(dai), joiner.addr, DAI_LP_AMOUNT);
+    deal(address(weth), joiner.addr, WETH_LP_AMOUNT);
+    deal(address(wbtc), joiner.addr, WBTC_LP_AMOUNT);
+    deal(address(pool), exiter.addr, ONE_UNIT, false);
 
     dai.approve(address(pool), type(uint256).max);
     weth.approve(address(pool), type(uint256).max);
+    wbtc.approve(address(pool), type(uint256).max);
     pool.bind(address(dai), DAI_LP_AMOUNT, 8e18); // 80% weight
     pool.bind(address(weth), WETH_LP_AMOUNT, 2e18); // 20% weight
+    pool.bind(address(wbtc), WBTC_LP_AMOUNT, 10e18); // +100% weight (unused in swaps)
```

Comment:
now it's 40% 10% 10%

Reply:
nee, is indistinct, because we're never swapping WBTC, from DAI-WETH perspective pool is `80-20`